docs/how-to-guides/new-user-guides/kubernetes-cluster-setup/kubernetes-cluster-setup.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: "Don't have a Kubernetes cluster? Try one of these tutorials."
+title: Kubernetes Cluster Setup
```

Comment:
```suggestion
title: Setting up a Kubernetes Cluster to Install Rancher
```

`Kubernetes Cluster` by itself is ambiguous (local vs downstream) so we shouldn't match what the current sidebar label has.

My suggestion is long for a sidebar label though so other choices are welcomed.

sidebar needs to be updated to match the final choice.

Reply:
What if we don't include "Kuberntes" in the label? Setting up a Cluster to Install Rancher. Setting up a Cluster for Rancher Server.

Reply:
Just "cluster" by itself has similar problems because we have pages on setting up the [infra](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/infrastructure-setup) that the K8s distros in this section are installed to.

We end up with cases where we use "cluster" to mean linux/windows nodes, linux/windows nodes with K8s distro (RKE/RKE2/K3s) or in this cases just setting up the K8s distro. With context it's fine, but there isn't much context to work off of when it's a title in the sidebar like here.

Reply:
@btat "Setting up a Kubernetes Cluster for Rancher Server" takes up 2 lines at 1920x1080 resolution. Imho 2 lines is ok, though one is preferred. 

![image](https://github.com/rancher/rancher-docs/assets/19174201/2b1759fe-a4ad-4271-9025-b0584bf34462)

Reply:
@martyav Agreed on 2 lines being ok since we don't have a good solution to clearly convey it with 1.